docs/deployments/free5gc-ulcl/eupf-b.yaml

```diff
@@ -0,0 +1,258 @@
+---
+
+image:
+  registry: ghcr.io
+  repository: edgecomllc
+  name: eupf
+  tag: main
+  pullPolicy: Always
+
+deploymentStrategy:
+  type: Recreate
+
+command:
+  - /bin/sh
+  - -c
+  - |
+      set -x ; mkdir /etc/iproute2 ; echo "111 fromn3gtp" >> /etc/iproute2/rt_tables ; \
+      ip rule add from 10.1.0.0/16 lookup fromn3gtp; \
+      ip route add default via 10.100.100.1 dev n6 table fromn3gtp ;\
+      sysctl -w net.ipv4.ip_forward=1  # to overcome the problem of switched off net.ipv4.ip_forward appearing in some environments ;\
+      /app/bin/eupf $0 $@
+
+args:
+  - --config
+  - /app/conf/config.yml
+
+securityContext:
+  privileged: true
+
+podSecurityContext:
+  sysctls:
+  - name: net.ipv4.ip_forward
+    value: "1"
+  - name: net.ipv6.conf.all.forwarding
+    value: "1"
+
+configMaps:
+  config:
+    data:
+      config.yml: |
+        interface_name: [n3, n6]
+        api_address: :8080
+        pfcp_address: 10.100.50.241:8805
+        metrics_address: :9090
+        n3_address: 10.100.50.233
+
+env:
+  UPF_PFCP_NODE_ID: 10.100.50.241  # address on n4 interface
+  # UPF_N3_ADDRESS: 10.100.50.233
+
+volumes:
+  - name: sys
+    hostPath:
+      path: /sys
+  - name: config
+    configMap:
+      name: edgecomllc-eupf-config
+
+volumeMounts:
+  - name: sys
+    mountPath: /sys
+    readOnly:  true
+  - name: config
+    mountPath: /app/conf
+
+metrics:
+  enabled: true
+  serviceMonitor:
+    enabled: true
+    additionalLabels:
+      release: kube-prometheus-stack
+    endpoints:
+      port: metrics
+      path: "/metrics"
+
+imagePullSecrets:
```

Comment:
we used it, when our repo was a private, now its public and our users can download docker image without this secret